### PR TITLE
Update pom.xml version on release

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

File: .travis/build.sh [hunk 1]
```diff
@@ -10,6 +10,7 @@ SYSTEMTEST_ARGS=${SYSTEMTEST_ARGS:-SmokeTest}
 
 if [ "$VERSION" != "latest" ]; then
     TAG=$VERSION
+    mvn versions:set -DnewVersion=$VERSION
```
Comment (rgodfrey) [inline .travis/build.sh]: Does the change to the .pom files actually get committed (and tagged) in git?

Comment (lulf) [reply .travis/build.sh]: No, I'm not sure we want to do that

File: release-scripts/update-snapshot-version.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+VERSION=$1
+
+git checkout master
+
+mvn versions:set -DnewVersion=${VERSION}
+
```
Comment (rgodfrey) [inline release-scripts/update-snapshot-version.sh]: don't we need to actually commit to git here?

Comment (lulf) [reply release-scripts/update-snapshot-version.sh]: Fixed!

**Pull request merged**